modules/sdk-unified-wallet/package.json

```diff
@@ -14,10 +14,11 @@
   },
   "dependencies": {
     "@bitgo/sdk-core": "^5.2.0",
-    "@bitgo/statics": "^14.1.0"
+    "@bitgo/statics": "^14.1.0",
+    "bitgo": "^16.4.0",
+    "openpgp": "5.1.0"
   },
   "devDependencies": {
-    "@bitgo/sdk-api": "^1.9.7",
```

Comment:
you're going to need this back in so you can write your unit tests without the `bitgo` module